### PR TITLE
Fast RTPS release 1.7.2 adoption [4832]

### PR DESCRIPTION
**Pull request by julianbermudez**
This pull request adapts the Agent to the new 1.7.2 release of Fast RTPS.
In particular, the CLI of the Agent has been modified in order to allow the load of a "reference file" different to the default one using previously.

Please, before reviewing this pull request, accept the following pull request:
- [x] [Documentation](https://github.com/eProsima/Micro-XRCE-DDS-docs/pull/31)

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro%20XRCE-DDS%20Agent%20Manual%20Linux/118/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro%20XRCE-DDS%20Agent%20Manual%20Windows/122/)

File: microxrce_agent.cpp [hunk 1]
```diff
@@ -169,23 +240,51 @@ int main(int argc, char** argv)
...
-                    if (0 == tcsetattr(fd, TCSANOW, &tty_config))
+                /*
+                 * Parse Refs File.
+                 */
+                if ((cl_counter <= cl.size()) && ("--refs" == cl[cl_counter - 1]))
+                {
+                    cl_counter++;
+                    if (cl_counter <= cl.size())
+                    {
+                        server->load_config_file(cl[cl_counter -1]);
```
Comment (BorjaOuterelo) [inline microxrce_agent.cpp]: Shall we do something with the returned bool?

File: microxrce_agent.cpp [hunk 2]
```diff
@@ -200,17 +299,44 @@ int main(int argc, char** argv)
...
-                    if (0 == tcsetattr(fd, TCSANOW, &attr))
+                /*
+                 * Parse Refs File.
+                 */
+                if ((cl_counter <= cl.size()) && ("--refs" == cl[cl_counter - 1]))
+                {
+                    cl_counter++;
+                    if (cl_counter <= cl.size())
+                    {
+                        server->load_config_file(cl[cl_counter -1]);
```
Comment (BorjaOuterelo) [inline microxrce_agent.cpp]: Shall we do something with the returned bool?

File: microxrce_agent.cpp (hunk 1 above)
Comment (julianbermudez) [reply microxrce_agent.cpp]: Fixed!

File: microxrce_agent.cpp (hunk 2 above)
Comment (julianbermudez) [reply microxrce_agent.cpp]: Fixed!

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro%20XRCE-DDS%20Agent%20Manual%20Linux/119/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro%20XRCE-DDS%20Agent%20Manual%20Windows/123/)

**Pull request merged**